### PR TITLE
Creating a stub of the file for Wynnewood PA

### LINKED ISSUE #38: Create a new file for kjell1979 hometown
## Add file for Wynnewood, PA
- [ ] Create a branch
- [ ] Add my file

### PR DESCRIPTION
**Pull request by kjell1979**
This is an example extended description to see how it looks like in github. Used for training purposes.

This Pull Request closes #38

### CODE REVIEW

File: wynnewood.pa [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Wynnewood, PA
+
+### Great places to eat
```
Comment (kjell1979) [inline wynnewood.pa:3]: :+1: here is... ?

**Comment by sedona2000** [comment]
hi, how is the weather?

File: wynnewood.pa (hunk 1 above)
Comment (pmitchell40) [inline wynnewood.pa:3]: @kjell1979 where are good places to eat in Wynnewood?

**Comment by pmitchell40** [comment]
@sedona2000 it is too cold for spring

**Comment by pmitchell40** [comment]
looks good @sedona2000 @kjell1979   ship it :shipit:

**Pull request merged**